easybuild/framework/easyblock.py

```diff
@@ -2435,8 +2435,12 @@ def build_and_install_one(ecdict, init_env):
             # collect build stats
             _log.info("Collecting build stats...")
 
-            buildstats = get_build_stats(app, start_time, build_option('command_line'))
-            _log.info("Build stats: %s" % buildstats)
+            if build_option('module_only'):
```

Comment:
`--skip --force` can be used to install additional extensions added in the easyconfig file, without having the reinstall the whole shebang, cfr. http://easybuild.readthedocs.io/en/latest/Partial_installations.html#partial-installation-skip